demo/src/plot_demo.rs

```diff
@@ -1101,6 +1107,61 @@ impl ChartsDemo {
     }
 }
 
+#[derive(PartialEq, serde::Deserialize, serde::Serialize, Default)]
+struct UserdataDemo {}
```

Comment:
It's soon time to break up this file into several smaller files (not in this PR though!)